integration-tests/pages/checkAnswersPage.js

```diff
@@ -5,45 +5,39 @@ const clickSubmit = () => cy.get('[data-submit]').click()
 export default () =>
   page('Check your answers before submitting the report', {
     verifyInputs: () => {
-      cy.get('.govuk-grid-column-full > :nth-child(2) > :nth-child(2) > :nth-child(2)').contains('Norman Smith')
-      cy.get('.govuk-grid-column-full > :nth-child(2) > :nth-child(3) > :nth-child(2)').contains('A1234AC')
```

Comment:
we still have an outstanding task to move these to use data-qa tags, so will be done as a follow up